common/predictive-text/package.json

```diff
@@ -1,14 +1,13 @@
 {
-  "name": "keyman-lmlayer-prototype",
-  "version": "0.0.0",
+  "name": "keyman-lmlayer",
+  "version": "14.0.0",
```

Comment:
👍

Reply:
Is this version number static? Or is it kept in sync with VERSION.md?

Reply:
Static, for now.

Reply:
I just had a major breakthrough on this.

`lerna version --amend <new-version-here> --yes --force-publish`

- [`--amend`](https://github.com/lerna/lerna/tree/master/commands/version#--amend): make changes locally, do not commit.  (Docs even state this is intended for CI)
- [`--yes`](https://github.com/lerna/lerna/tree/master/commands/version#--yes): do not prompt, assume we'll say 'yes' for all prompts.
    - so, this can be safely omitted in non-CI environments; useful for testing/learning the command
    - I'm... fairly certain the example is a copy from the `lerna publish --yes` documentation.
- [`--force-publish`](https://github.com/lerna/lerna/tree/master/commands/version#--force-publish): do not check `git diff` - assume every package changed.  Doesn't actually publish.
    - Not sure exactly why it's named this; there's no matching option on `lerna publish`.
    - In isolation, `lerna version` likes to automatically `git push` the changes to the repository.  So, "publish" here would be "to Github", not "to `npm`."

<img width="697" alt="lerna versioning" src="https://user-images.githubusercontent.com/25213402/80331377-1ea4fd80-8872-11ea-9525-b5678e2a22c1.png">

Note:
- `Skipping git push`
- `Skipping releases`

----

So, once `lerna` hits `master`, it may be worth adding this call to our daily version-update Action.  We can finally have auto-updated version info in our repo's package.json files!  (Well, the ones managed by `lerna`, anyway.)

Reply:
Good find!

But oh my gosh I really dislike the opinionated my-way-or-the-highway defaults of NPM ... having to specify --no-git-tag-version to disable default functionality is just SO BACKWARDS!

Reply:
Yeah, I feel you.  _This_ one's on `lerna`; it tries to do everything out-of-the-box for version management.

I've now added an appropriate script definition to the base package:  I've converted the above into a simple `npm run version 14.xx.xx` call at the repo's base.